hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3ARetryPolicy.java

```diff
@@ -80,13 +82,18 @@
 @SuppressWarnings("visibilitymodifier")  // I want a struct of finals, for real.
 public class S3ARetryPolicy implements RetryPolicy {
 
+  private static final Logger LOG = LoggerFactory.getLogger(
+      S3ARetryPolicy.class);
+
+  private final Configuration configuration;
+
   /** Final retry policy we end up with. */
   private final RetryPolicy retryPolicy;
 
   // Retry policies for mapping exceptions to
 
   /** Base policy from configuration. */
-  protected final RetryPolicy fixedRetries;
+  protected final RetryPolicy defaultRetries;
```

Comment:
maybe exponentialBackoffRetry would be a better name for this, or do you plan to make the retry implementation pluggable here?

Reply:
now `baseExponentialRetry`